packages/babel-plugin/src/__tests__/expression-evaluation.test.tsx

```diff
@@ -23,7 +19,7 @@ describe('import specifiers', () => {
       <div css={{ fontSize: 8 * 2 }}>hello world</div>
     `);
 
-    expect(actual).toInclude('.cc-hash-test{font-size:16px}');
```

Comment:
ahh finally!! we removed this hash mocking ❤️

Reply:
hehe yeah, figured it was time...